### DISCUSSION TITLE
Record video stream options

### DISCUSSION THREAD

**Discussion by briodan** [easytarget/esp32-cam-webserver#295] [Q&A]
Not sure if this belongs here or not but was wondering if anyone had any luck recording the video stream from this with an NVR or similar?

I have tried several options:
- BlueIris can view the camera but does not record anything
- Frigate can view the camera seems to record but the video file is corrupted
- MotionEye can view the camera but has no options to record

any suggestions/options?

**Comment by cor-of-org** [comment]
Have you tried VLC? I sometimes use it to record the stream from TL-CAM (running ESP32-CAM).
